intdataplane/int_dataplane.go

```diff
@@ -750,3 +752,39 @@ type iptablesTable interface {
 	RemoveChains([]*iptables.Chain)
 	RemoveChainByName(name string)
 }
+
+// msgStringer wraps an API message to customise how we stringify it.  For example, it truncates
+// the lists of members in the (potentially very large) IPSetsUpdate messages.
+type msgStringer struct {
+	msg interface{}
+}
+
+func (m msgStringer) String() string {
+	if log.GetLevel() < log.DebugLevel && m.msg != nil {
```

Comment:
Does `log.GetLevel()` give the level of the log that is currently being emitted, or the configured logging level?

If it's the latter, is that what you wanted?  I.e., if configured level is DEBUG, then we won't do any message truncation, even in INFO logs.  Seems fine to me FWIW - just checking.